docs/core/porting/index.md

```diff
@@ -1,116 +1,164 @@
+### .NET API Analyzer
 
-   ```csharp
-   #if FEATURE_APPDOMAIN_LOADING
-   // Code that uses appdomains
-   #elif FEATURE_ASSEMBLY_LOAD_CONTEXT
-   // Code that uses assembly load context
-   #else
-   #error Unsupported platform
-   #endif
-   ```
+The [.NET API Analyzer](../../standard/analyzers/api-analyzer.md) analyzes whether or not you're using an API that will throw a <xref:System.PlatformNotSupportedException> at run-time. Although this isn't common if you're moving from .NET Framework 4.7.2 or higher, it's good to check. For more information about APIs that throw exceptions on .NET, see [APIs that always throw exceptions on .NET Core](../compatibility/unsupported-apis.md).
 
-1. Install the [.NET API analyzer](../../standard/analyzers/api-analyzer.md) into your projects to identify APIs that throw <xref:System.PlatformNotSupportedException> on some platforms and some other potential compatibility issues.
+API Analyzer comes as a NuGet package [Microsoft.DotNet.Analyzers.Compatibility](https://www.nuget.org/packages/Microsoft.DotNet.Analyzers.Compatibility/) you add to your project.
 
-   This tool is similar to the portability analyzer, but instead of analyzing if code can build on .NET Core, it analyzes whether you're using an API in a way that will throw a <xref:System.PlatformNotSupportedException> at run time. Although this isn't common if you're moving from .NET Framework 4.7.2 or higher, it's good to check. For more information about APIs that throw exceptions on .NET Core, see [APIs that always throw exceptions on .NET Core](../compatibility/unsupported-apis.md).
+For more information, see [.NET API analyzer](../../standard/analyzers/api-analyzer.md).
 
-1. At this point, you can switch to targeting .NET Core (generally for applications) or .NET Standard (for libraries).
+## Considerations when porting
 
-   The choice between .NET Core and .NET Standard is largely dependent on where the project will be run. If it is a library that will be consumed by other applications or distributed via NuGet, the preference is usually to target .NET Standard. However, there may be APIs that are only available on .NET Core for performance or other reasons; if that's the case, .NET Core should be targeted with potentially a .NET Standard build available as well with reduced performance or functionality. By targeting .NET Standard, the project will be ready to run on new platforms (such as WebAssembly). If the project has dependencies on specific app frameworks (such as ASP.NET Core), then the target will be limited by what the dependencies support.
+When porting your application to .NET, consider the following suggestions.
```

Comment:
Consider re-ordering this list so the DO recommendations are all first. That way, readers see the most important tasks first.

Reply:
I missed your ordering. It does make sense. So, maybe add a sentence that the list is in the order to perform your tasks?